4.3: VM deployment adjustments (#3343)
uyuni-project/uyuni-docs: modules/installation-and-upgrade/pages/install-vm.adoc

> This chapter provides procedures how you can deploy a Kernel Virtual Machine (KVM) or VMware image.

Reviewer: ```suggestion
This chapter provides procedures for deploying a Kernel Virtual Machine (KVM) or VMware image.
```

| VirtIO Disk 3        | pgsql     | 100{nbsp}GB
| VirtIO Disk 4        | swap      | 4{nbsp}GB
|===
+

. Click btn:[Begin Installation] to boot the new VM from the {productname} image.

> . At the JeOS Firstboot screen select start to continue and check the following configuration dialogs (keyboard layout, license agreement, time zone, password for root).

Reviewer: ```suggestion
. At the JeOS Firstboot screen select [literal]``start`` to continue and check the following configuration options (keyboard layout, license agreement, time zone, password for root).
```

> . When installation completes log in as root.

Reviewer: ```suggestion
. When installation completes, log in as root.
```

. Proceed with xref:installation-and-upgrade:install-vm.adoc#minimal.susemgr.prep[].

// FIXME: check the following and esp. replace with firstboot (see 5.0 docs)
== {productname} Virtual Machine Creation - VMware

This sections describes VMware configurations, focusing on the creation of an extra virtual disk essential for the {productname} storage partition within VMware environments.

.Procedure: Creating the VMware Virtual Machine

. Download {productname} Server [filename]``.vmdk`` file.
  Then transfer a copy to your VMware storage.

. Make a copy of uploaded [filename]``.vmdk`` file using VMware web interface.

>   This will convert provided [filename]``.vmdk`` file to the format suitable for vSpere hypervisor.

Reviewer 2: Maybe add clarification `Use this new copy as a base image for the virtual machine`

. Create and name a new virtual machine based on the Guest OS Family [systemitem]``Linux`` and Guest OS Version {sle} 15 (64-bit).
+

// FIXME: Do we need the following step?  I think, in 5.0 we removed it

> . In [guimenu]``Customize settings`` browse to the uploaded [filename]``.vmdk`` using [systemitem]``IDE controller 0`` storage device in the [systemitem]``Controller Location``.

Reviewer 2: Not needed, please remove.
